README.md

```diff
@@ -24,19 +24,19 @@ These tags have image version postfix. This updates many places so we need feedb
 
 Current images use fluentd v1 series.
 
-- `v1.14.0-1.1`, `v1.14-1`, `edge`
+- `v1.14.4-1.0`, `v1.14-1`, `edge`
   [(v1.14/alpine/Dockerfile)][fluentd-1-alpine]
-- `v1.14.0-debian-1.1`, `v1.14-debian-1`, `edge-debian`
+- `v1.14.4-debian-1.1`, `v1.14-debian-1`, `edge-debian`
   [(v1.14/debian/Dockerfile)][fluentd-1-debian]
-- `v1.14.0-debian-arm64-1.1`, `v1.14-debian-arm64-1`, `edge-debian-arm64`
+- `v1.14.4-debian-arm64-1.1`, `v1.14-debian-arm64-1`, `edge-debian-arm64`
   [(v1.14/arm64/debian/Dockerfile)][fluentd-1-debian-arm64]
```

Comment:
It should be v1.14.4-debian-arm64-1.0

Reply:
Ditto. Updated by 2825eac33e63dab24c7b8ca3aacea18b199c3204.